<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clsTraceSetup</class>
 <widget class="QDialog" name="clsTraceSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>扫描设置</string>
  </property>
  <property name="windowIcon">
   <iconset resource="myIcon.qrc">
    <normaloff>:/Icon/Icons/wk.ico</normaloff>:/Icon/Icons/wk.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	
	font: 75 12pt &quot;楷体&quot;;
	background-color: rgb(148, 148, 248);
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;楷体&quot;;</string>
     </property>
     <property name="title">
      <string>设置偏置点</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" rowspan="2">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="btnBias1">
          <property name="minimumSize">
           <size>
            <width>88</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>88</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="btnBias2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>88</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="btnBias3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>88</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="btnBias4">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>88</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="btnBias5">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>88</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1" rowspan="2" colspan="2">
       <widget class="Line" name="line">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="btnAdd">
          <property name="minimumSize">
           <size>
            <width>88</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>88</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>添加</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>28</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnSet">
          <property name="minimumSize">
           <size>
            <width>88</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>88</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>设置</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>28</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnDel">
          <property name="minimumSize">
           <size>
            <width>88</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>88</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>删除</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_9">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>28</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnSort">
          <property name="minimumSize">
           <size>
            <width>88</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>88</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>排序</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;楷体&quot;;</string>
       </property>
       <property name="title">
        <string>温度设置</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="maximumSize">
             <size>
              <width>78</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>温度上限：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnTemp">
            <property name="minimumSize">
             <size>
              <width>88</width>
              <height>36</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>88</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>50</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="maximumSize">
             <size>
              <width>78</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>延迟时间：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDelay">
            <property name="minimumSize">
             <size>
              <width>88</width>
              <height>36</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>88</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>50</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_7">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="btnCancel">
         <property name="minimumSize">
          <size>
           <width>88</width>
           <height>36</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>88</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;楷体&quot;;</string>
         </property>
         <property name="text">
          <string>关闭</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnOK">
         <property name="minimumSize">
          <size>
           <width>88</width>
           <height>36</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>88</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;楷体&quot;;</string>
         </property>
         <property name="text">
          <string>确认</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="myIcon.qrc"/>
 </resources>
 <connections/>
</ui>
